contents/swift/welcome-to-swift.md

```diff
@@ -12,44 +12,46 @@
+Introduced in 2014, Swift has concise and expressive syntax, compared with its predecessor language **Objective-C**. It also supports playgrounds, an innovative feature that allows programmers to experiment with Swift code and see the results immediately, without the overhead of building and running an app.
 
-# Getting Started
+## Why should I learn Swift?
 
-Reading the [Language Guide in the official documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is definitely recommended, since it explains everything quite clearly, albeit quite verbose.
+1. **iOS developers have good employment.** Mobile development becomes popular recently. A [survey in StackOverFlow](https://insights.stackoverflow.com/survey/2017#developer-profile-developer-type) shows that mobile development, although is significantly less than web development, still occupies large marketing along with desktop application development. In addition, iOS developer is well-paid. The [salary statistics](https://www.indeed.com/salaries/iOS-Developer-Salaries) shows that iOS developer in US can earn more than $120K per year. Therefore, learning Swift is helpful for your career.
 
-If you're just looking to take a dip into Swift, here are the [basics](https://guides.codepath.com/ios/Swift-Basics) you'll need to understand.
+1. **Swift is easy to learn.** Swift is aimed to be designed as a simple and understandable language by Apple. Its syntax resembles English, and thus is easy to read and understand. Any programmer being familiar with Python, Java or C++ can learn Swift quickly. With its simplistic and readable syntax, learning Swift is easy even for those just starting out with programming!
 
-And if you are really pressed for time, here's a couple of cheatsheets with code examples:
+1. **Swift is an [open source project](https://github.com/apple/swift).** Learning about Swift and contributing to the language may be a useful experience for those interested. In addition, its source code may be useful reading for those interested in learning good practices, or simply to find out more about how the language was implemented.
 
-- [Swift Cheat Sheet](https://mhm5000.gitbooks.io/swift-cheat-sheet/content/index.html) (web-friendly)
-- [Swift 3.0 Cheat Sheet and Quick Reference](https://koenig-media.raywenderlich.com/uploads/2014/06/RW-Swift-Cheatsheet-0_7.pdf) (print-friendly)
+Also note that Swift is not perfect. It is still very young, and therefore the language itself may have issues to be addressed, such as [stored properties in extension](https://medium.com/@valv0/computed-properties-and-extensions-a-pure-swift-approach-64733768112c). This may suffer the programmers during development.
+In addition, Swift still has limited number of libraries and tools, and many of the resources dedicated to earlier Swift versions could be useless with the newer releases.
```

Comment:
In addition, Swift's libraries/tools are still limited and resources dedicated to earlier Swift versions may not be usable with newer releases.